<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FitsHeaderModifierDialog</class>
 <widget class="QDialog" name="FitsHeaderModifierDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Type the values for the following keywords.&lt;/p&gt;&lt;p&gt;A new file will be created.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="boxCTYPE">
       <property name="title">
        <string>CTYPE</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QLineEdit" name="textCTYPE1">
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="textCTYPE2">
          <property name="text">
           <string>y</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>CTYPE1</string>
          </property>
          <property name="buddy">
           <cstring>textCTYPE1</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>CTYPE2</string>
          </property>
          <property name="buddy">
           <cstring>textCTYPE2</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>CTYPE3</string>
          </property>
          <property name="buddy">
           <cstring>textCTYPE3</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="textCTYPE3">
          <property name="text">
           <string>z</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="boxCUNIT">
       <property name="title">
        <string>CUNIT</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="4" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>CUNIT3</string>
          </property>
          <property name="buddy">
           <cstring>textCUNIT3</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>CUNIT1</string>
          </property>
          <property name="buddy">
           <cstring>textCUNIT1</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="textCUNIT2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>CUNIT2</string>
          </property>
          <property name="buddy">
           <cstring>textCUNIT2</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="textCUNIT3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="textCUNIT1">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="boxCDELT">
       <property name="title">
        <string>CDELT</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>CDELT1</string>
          </property>
          <property name="buddy">
           <cstring>textCDELT1</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="textCDELT1"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>CDELT2</string>
          </property>
          <property name="buddy">
           <cstring>textCDELT2</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="textCDELT2"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>CDELT3</string>
          </property>
          <property name="buddy">
           <cstring>textCDELT3</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="textCDELT3"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>textCTYPE1</tabstop>
  <tabstop>textCDELT1</tabstop>
  <tabstop>textCDELT2</tabstop>
  <tabstop>textCDELT3</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FitsHeaderModifierDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FitsHeaderModifierDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textCUNIT1</sender>
   <signal>textChanged(QString)</signal>
   <receiver>textCUNIT2</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textCUNIT2</sender>
   <signal>textChanged(QString)</signal>
   <receiver>textCUNIT3</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textCUNIT3</sender>
   <signal>textChanged(QString)</signal>
   <receiver>textCUNIT1</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
